### PR TITLE
Add missing HDCamera/UniversalCamera component (ABC-364)

### PR DESCRIPTION
**Pull request by thomas-tu**
# Purpose of this PR:

ABC-364

Imported Cameras from Alembic don't have the additional components HDRP and URP use. They have to be manually added to fully work with their respective rendering pipeline.

# Testing

Added an automatic test with an asset from @cguer for HDRP and URP.

### CODE REVIEW

**Review by RoseHirigoyen — APPROVED** [review]

**Review by arejayelle — APPROVED** [review]

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2023.1.0b2 (Windows only, HDRP + URP + Legacy)

**Review by lucillecaillaud — COMMENTED** [review]
Code looks good to me!

Waiting on more info on [this thread](https://unity.slack.com/archives/C026KK6FCMB/p1676301354805589) to validate that it's the correct way to fix the issue.

Even though the problem is only reproducible with the "solutions recorder" it still sounds correct that when you import an Alembic in Unity it's correct straight away.

The only potential problem I'd see to that would be that it could impact import performances. But the difference is so small, I'm not sure how it could be noticeable.

**Review by lucillecaillaud — APPROVED** [review]

**Review by MaryPerchanok — CHANGES_REQUESTED** [review]

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [2.3.1] - 2022-12-06
 ### Added
+- Comform the GameObject cameras to these rendering pipelines with the additional data at import time.
```
[the remark this answers is not in the document]
Comment (MaryPerchanok) [reply com.unity.formats.alembic/CHANGELOG.md:9]: Suggestion:

- At import, additional data is added to GameObject cameras to conform with URP or HDRP rendering pipelines.

**Review by MaryPerchanok — CHANGES_REQUESTED** [review]

**Review by MaryPerchanok — APPROVED** [review]
Approved!

**Pull request merged**